friends/src/App.js

```diff
@@ -1,28 +1,50 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import axios from 'axios';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
+export default class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friendsList: [],
+      name: "",
+      age: "",
+      email: ""
+    }
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:5000/friends').then(response => this.setState({friendsList: [...response.data]}))
```

Comment:
I recommend adding a catch statement to handle errors that may occur with the request.